stats/pp_stage_goal.php

```diff
@@ -0,0 +1,81 @@
+<?php
+$relPath = "./../pinc/";
+include_once($relPath.'base.inc');
+include_once($relPath.'stages.inc');
+include_once($relPath.'graph_data.inc');
+include_once($relPath.'slim_header.inc');
+include_once($relPath.'page_tally.inc');
+
+// Get start of current month as unixtime
+$m_start = mktime(0, 0, 0, date('m'), 1, date('Y'));
+
+// We want the "PP pages goal" to be equal to the current month's F2 actuals
+$page_offset = 0;
+$site_stats = get_site_page_tally_summary("F2");
+$pp_page_goal = $site_stats->curr_month_actual + $page_offset;
+
+// Start with creating the Graph, this enables the use of the cache
```

Comment:
This comment is no longer true and can be removed. We could probably move the width/height definitions down to just before we define `$js_data`.